src/lib/php/Dao/UploadDao.php

```diff
@@ -101,6 +101,15 @@ public function getUpload($uploadId)
     return $row ? Upload::createFromTable($row) : null;
   }
 
+  public function getActiveUploadsArray()
+  {
+    $stmt = __METHOD__;
+    $result = $this->dbManager->getRows("SELECT * FROM upload where pfile_fk IS NOT NULL",
+        array(), $stmt);
+
```

Comment:
As a suggestion, how do you think about returning an array of [Upload](https://github.com/fossology/fossology/blob/b7a846ca391b091407b8173ec0f74a1dbab5bc46/src/lib/php/Data/Upload/Upload.php#L38) objects?

Reply:
Thanks for this tip - corected as you suggested